enforcer.go

```diff
@@ -610,6 +611,34 @@ func (e *Enforcer) EnforceExWithMatcher(matcher string, rvals ...interface{}) (b
 	return result, explain, err
 }
 
+// BatchEnforce enforce in batches
+func (e *Enforcer) BatchEnforce(requests [][]string) (map[string]bool, error) {
```

Comment:
requests should be `[][]interface`.

Reply:
If we change to `[][]interface{}`, we couldn't pass `[][]string{}` to this function.